tutorials/quickstart/tutorial.md

```diff
@@ -22,18 +22,18 @@ You'll also see how the additional tools in the Particular Service Platform make
 
 #variant Variant
 
-In this tutorial, you'll see how NServiceBus takes the grunt work out of system design by handling all of the plumbing for you.
+In this tutorial, you'll see how NServiceBus takes the grunt work out of system design by handling all the plumbing for you.
 
-You will explore part of an e-commerce system that handles receiving an order placed by a user, as well as the billing and shipping that goes along with it. This is what the final solution will look like:
+You'll explore part of an e-commerce system that handles receiving an order placed by a user, and the billing and shipping that goes with it. The final solution will look like this:
 
 ![quickstart solution diagram](/tutorials/quickstart/after.svg)
 
-The tutorial is composed of 3 parts - introduction, failure recovery, and extensibility - that demonstrate how NServiceBus helps with design best practices and how the Particular Service Platform helps monitor a distributed system. Each part takes approximately 15-20 minutes to complete.
+The tutorial has 3 parts—an introduction, failure recovery, and extensibility—that demonstrate how NServiceBus helps with best design practices and how the Particular Service Platform helps monitor a distributed system. Each part takes about 15-20 minutes to complete.
 
-To run this tutorial you’ll need:
+To follow this tutorial you’ll need:
 
-- Experience with C# coding
-- Windows 10 or 11 running an IDE for C# code, for example Visual Studio 2022, Visual Studio Code, or JetBrains Rider.
+- Experience with C#
```

Comment:
There's not much else about C# other than _coding_, so the word seems redundant.